src/Symfony/Component/Validator/Tests/Fixtures/Attribute/Nested/Entity.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Fixtures\Attribute\Nested;
+
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Tests\Fixtures\Attribute\EntityParent;
+use Symfony\Component\Validator\Tests\Fixtures\EntityInterfaceB;
+use Symfony\Component\Validator\Tests\Fixtures\CallbackClass;
+use Symfony\Component\Validator\Tests\Fixtures\ConstraintA;
+
+#[
+    ConstraintA,
+    Assert\GroupSequence(['Foo', 'Entity']),
+    Assert\Callback([CallbackClass::class, 'callback']),
+]
+class Entity extends EntityParent implements EntityInterfaceB
+{
+    #[
+        Assert\NotNull,
+        Assert\Range(min: 3),
+        Assert\All([
+            new Assert\NotNull(),
+            new Assert\Range(min: 3),
+        ]),
+        Assert\All([
+            'constraints' => [
+                new Assert\NotNull(),
+                new Assert\Range(min: 3),
+            ],
+        ]),
```

Comment:
```suggestion
        Assert\All(
            constraints: [
                new Assert\NotNull(),
                new Assert\Range(min: 3),
            ],
        ),
```

Let's switch to named arguments, like we did for the other constraints.

Reply:
Looking into it!

Reply:
I set `mixed` as type of constructor parameter, because of this:

https://github.com/symfony/symfony/blob/e34cd7dd2c6d0b30d24cad443b8f964daa841d71/src/Symfony/Component/Validator/Tests/Constraints/AllValidatorTest.php#L30

Reply:
Didn't type-hinted the constructor parameter because of this:

https://github.com/symfony/symfony/blob/e34cd7dd2c6d0b30d24cad443b8f964daa841d71/src/Symfony/Component/Validator/Tests/Constraints/AllValidatorTest.php#L30

Reply:
Done!